### PR TITLE
Remove clamp from CoreMath.h

### PR DESCRIPTION
**Pull request by beckerhe**
This function can be replaced by std::clamp.

### CODE REVIEW

**Review by dimitry- — APPROVED** [review]

File: src/OrbitGl/CoreMath.h [hunk 1]
```diff
@@ -21,9 +21,4 @@ using Vec2i = gte::Vector2<int>;
 using Vec3i = gte::Vector3<int>;
 using Vec4i = gte::Vector4<int>;
 
-using Color = gte::Vector4<unsigned char>;
-
-template <class T>
-constexpr const T& clamp(const T& v, const T& lo, const T& hi) {
-  return std::min(std::max(v, lo), hi);
-}
+using Color = gte::Vector4<unsigned char>;
```
Comment (dimitry-) [inline src/OrbitGl/CoreMath.h:24]: seems like this file is missing include guard?

Comment (beckerhe) [reply src/OrbitGl/CoreMath.h:24]: Hmm yes. You're right. I didn't even notice that. I will add them.

Comment (beckerhe) [reply src/OrbitGl/CoreMath.h:24]: Ah. It has `pragma once`. I will change it to include guards

Comment (beckerhe) [reply src/OrbitGl/CoreMath.h]: done.

**Pull request merged**